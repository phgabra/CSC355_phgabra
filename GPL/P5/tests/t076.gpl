int i1 = 4 + -46;
int i2 = 40 - -2;
int i3 = -2 + 44;
int i4 = -2 + -40;
int i5 = - - 2 - -40;

double x1 = 4 + -46;
double x2 = 40 - -2;
double x3 = -2 + 44;
double x4 = -2 + -40;
double x5 = - - 2 - -40;

double y1 = 4 + -46.1;
double y2 = 40.1 - -2;
double y3 = -2 + 44.1;
double y4 = -2.1 + -40;
double y5 = - - 2.1 - -40;
