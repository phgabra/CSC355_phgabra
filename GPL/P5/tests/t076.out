gpl.cpp::main()
  input file(tests/t076.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int i1 = -42
int i2 = 42
int i3 = 42
int i4 = -42
int i5 = 42
double x1 = -42
double x2 = 42
double x3 = 42
double x4 = -42
double x5 = 42
double y1 = -42.1
double y2 = 42.1
double y3 = 42.1
double y4 = -42.1
double y5 = 42.1
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
